config/PolyDbCfg.fpp

```diff
@@ -0,0 +1,34 @@
+# ======================================================================
```

Comment:
Exactly how I'd have done this.  Good pattern!

Reply:
![image](https://github.com/nasa/fprime/assets/8086374/2aac781f-af0e-4866-8d6d-2a8d9e218b87)